jck/jtrunner/JavatestUtil.java

```diff
@@ -534,37 +546,47 @@ else if ( !tests.contains("api/java_awt/interactive") ) {
 			}
 			
 			fileContent += "concurrency " + concurrencyString + ";\n";
-                       
-                        if (!timeoutFactorString.equals("NULL")) {
-				fileContent += "timeoutfactor " + timeoutFactorString + ";\n";
-                        } else {
-				fileContent += "timeoutfactor 4" + ";\n";	// 4 base time limit equal 40 minutes
+			generatedJti.setProperty("concurrency", concurrencyString);
+            if (timeoutFactorString.equals("NULL")) {
+				timeoutFactorString = "4";  // 4 base time limit equal 40 minutes
 			}
+			fileContent += "timeoutfactor " + timeoutFactorString + ";\n";
+			generatedJti.setProperty("timeoutfactor", timeoutFactorString);
+            
 			fileContent += keyword + ";\n";
+			generatedJti.setProperty(keyword, "");
```

Comment:
I think "keyword" is generic all tests?

Also, "i think" some of the jti property key names differ, for example "keyword" in the jti file is:
```
jck.keywords.keywords.mode=expr
jck.keywords.keywords.value=\!interactive
jck.keywords.keywords=\!interactive
jck.keywords.needKeywords=Yes
```
ref: https://github.com/adoptium/aqa-tests/blob/master/jck/jtrunner/config/jck24/runtime.jti
whereas in jtb is:
```
keywords !interactive;
```

I suspect however, the ones where this differs are probably "generic"(all tests) ones which we can leave in JTB ?
ie.only need to write the "set jck.XXXXX ..." ones to jti maybe?